Azure/azure-cli
# [AppService] functionapp: Fixed configuration of container settings on linux dotnet function apps

Resolves #12773 
This changes how linuxFxVersion is configured for linux dotnet function apps. Right now, the cli is incorrectly configuring this value and linux consumption apps are failing to be created. This should fix that issue.

**Testing Guide**  
A unit test was created for this: -- 
```
azdev test appservice test_functionapp_on_linux_dotnet_consumption`.
```

You can also ensure that the following command can create an app successfully --
```
az functionapp create --name name -g resource_group -s storage_account -c location --os-type linux --runtime dotnet
```


**History Notes:**  
[AppService] az functionapp create: Changed the way linuxFxVersion was being set for dotnet linux function apps. This should fix a bug that was preventing dotnet linux consumption apps from being created. 

---

This checklist is used to make sure that common guidelines for a pull request are followed.
